<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfRingDataDialog</class>
 <widget class="QDialog" name="WkfRingDataDialog">
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="minValueLabel">
     <property name="text">
      <string>Min Value:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="minValueBox">
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="maxValueLabel">
     <property name="text">
      <string>Max Value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="maxValueBox">
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="minValueColorLabel">
     <property name="text">
      <string>Minimum Value Color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="UtQtColorButton" name="minValueColor">
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="maxValueColorLabel">
     <property name="text">
      <string>Maximum Value Color:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="UtQtColorButton" name="maxValueColor">
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="clampToBaseBox">
     <property name="text">
      <string>Clamp to Baseline</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
</ui>
